rpcs3/Emu/Cell/lv2/sys_lwcond.cpp

```diff
@@ -290,12 +290,6 @@ error_code _sys_lwcond_queue_wait(ppu_thread& ppu, u32 lwcond_id, u32 lwmutex_id
 
 				cond->waiters--;
 
-				if (mutex->signaled.fetch_dec_sat())
```

Comment:
What problems was it causing?
The change itself is acceptable though.

Reply:
This commit originated from glances at the syscall in ida, so I've tested this using direct lv2 calls to test the mutex state after the timeout. eg the code below would generate correct results with this change.
```
inline bool test_lock(sys_lwmutex_t* mtx)
{
	if (_sys_lwmutex_trylock(mtx) == CELL_OK) 
	{
		_sys_lwmutex_unlock(mtx);
		return false;
	}
	return true;
}
```

Reply:
those are syscall macros from the testcase code.